starter/src/app/home/home.component.ts

```diff
@@ -77,33 +81,31 @@ export class HomeComponent implements OnInit {
 		let httpRequest$: Observable<StarkSingleItemResponseWrapper<Request> | StarkCollectionResponseWrapper<Request>>;
 
 		if (type === "singleItem") {
-			httpRequest$ = this.httpService
-				.executeSingleItemRequest({
-					backend: backend,
-					resourcePath: "requests/11",
-					headers: new Map<string, string>(),
-					queryParameters: new Map<string, string | string[] | undefined>(),
-					requestType: StarkHttpRequestType.GET,
-					serializer: serializer,
-					retryCount: 4
-				})
+			httpRequest$ = this.httpService.executeSingleItemRequest({
+				backend: backend,
+				resourcePath: "requests/11",
+				headers: new Map<string, string>(),
+				queryParameters: new Map<string, string | string[] | undefined>(),
+				requestType: StarkHttpRequestType.GET,
+				serializer: serializer,
+				retryCount: 4
+			});
 		} else {
-			httpRequest$ = this.httpService
-				.executeCollectionRequest({
-					backend: backend,
-					resourcePath: "requests",
-					headers: new Map<string, string>(),
-					queryParameters: new Map<string, string | string[] | undefined>(),
-					requestType: StarkHttpRequestType.GET_COLLECTION,
-					serializer: serializer,
-					retryCount: 4
-				})
+			httpRequest$ = this.httpService.executeCollectionRequest({
+				backend: backend,
+				resourcePath: "requests",
+				headers: new Map<string, string>(),
+				queryParameters: new Map<string, string | string[] | undefined>(),
+				requestType: StarkHttpRequestType.GET_COLLECTION,
+				serializer: serializer,
+				retryCount: 4
+			});
 		}
 
 		httpRequest$.subscribe(
 			(response: any) => console.log("---------- SUCCESS", response),
-			(error: StarkHttpErrorWrapper) => console.log("---------- ERROR", error),
-			() => console.log("---------- COMPLETE")
+			(error: StarkHttpErrorWrapper) => this.loggingService.error("---------- ERROR", <any>error),
```

Comment:
Why do you cast the error as "any" instead of calling error.toString() or something like this ?

Reply:
Indeed, then we need to change our StarkHttpErrorWrapper so that it extends Error so we can keep the stacktrace and simply pass it to the logging service as is